source/Halibut/Transport/Protocol/RegisteredSerializationBinder.cs

```diff
@@ -6,17 +6,23 @@ namespace Halibut.Transport.Protocol
     public class RegisteredSerializationBinder : ISerializationBinder
     {
         readonly ISerializationBinder baseBinder = new DefaultSerializationBinder();
-        readonly Type[] protocolTypes = { typeof(ResponseMessage), typeof(RequestMessage) };
+        static readonly Type[] protocolTypes = { typeof(ResponseMessage), typeof(RequestMessage) };
         readonly ITypeRegistry typeRegistry;
 
         public RegisteredSerializationBinder() : this(new TypeRegistry())
         {
-        } // kept for backwards compatibility.
+            AddProtocolTupesToTypeRegistry(this.typeRegistry);
+        }
+
+        public static void AddProtocolTupesToTypeRegistry(ITypeRegistry typeRegistry)
+        {
+            foreach (var protocolType in protocolTypes) typeRegistry.RegisterType(protocolType, protocolType.Name, true);
```

Comment:
nit: if we're moving this anyway, let's format it with curly braces across multiple lines.

Reply:
ok